--- package/network/services/hostapd/Makefile
+++ package/network/services/hostapd/Makefile
@@ -10,7 +10,7 @@
 PKG_SOURCE_URL:=https://w1.fi/hostap.git
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_DATE:=2026-07-09
 PKG_SOURCE_VERSION:=f08f2749aa696c4e47c5c0f591dda99951bf9fac
-PKG_MIRROR_HASH:=skip
+PKG_MIRROR_HASH:=c1a978092c8a6e43a9e1bd47e1f334637b9c4969ca741c30ff45c24539132ca4
 
 PKG_MAINTAINER:=Felix Fietkau <nbd@nbd.name>
 PKG_LICENSE:=BSD-3-Clause
